styles.css

```diff
@@ -0,0 +1,73 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background: #191765;
+  font-family: 'Roboto', sans-serif;
+}
```

Comment:
- [ ] I also suggest that you add the background image as seen in the design you chose, it will add a sophisticated look to your portfolio.

### Your design
![Screenshot 2021-11-25 at 14-05-14 My Portfolio](https://user-images.githubusercontent.com/4247740/143430374-4b542bb5-9e2b-467e-830b-98a0136f16f8.png)


### Figma design
<img width="283" alt="cap" src="https://user-images.githubusercontent.com/4247740/143430785-68052b36-7558-4118-845c-a43062b6c0bb.PNG">